### PR TITLE
[DOC] Update core-devs list

### PR DESCRIPTION
**Pull request by NicolasGensollen**
Welcome @alexisthual ! :tada: 
PR also fixes the alphabetical order in core-dev list.

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -21,17 +21,18 @@ Core developers
...
 
 * Alexandre Gramfort
+* Alexis Thual
 * Bertrand Thirion
+* Binh Nguyen
 * Elizabeth DuPre
 * Gael Varoquaux
 * Jerome Dockes
 * Julia Huntenburg
 * KamalakerDadi
```
Comment (bthirion) [inline AUTHORS.rst:31]:
```suggestion
* Kamalaker Dadi
```

**Review by bthirion — COMMENTED** [review]
one small glitch

**Comment by GaelVaroquaux** [comment]
Excellent!! Welcome @alexisthual !

**Comment by NicolasGensollen** [comment]
Merging, thanks!

**Pull request merged**